server/cluster.go

```diff
@@ -386,10 +429,81 @@ func (c *Cluster) Master(msg *ClusterReq, rejected *bool) error {
 	return nil
 }
 
+func (c *Cluster) TopicMaster(msg *ClusterReq, rejected *bool) error {
+	*rejected = false
+
+	sid := "topic-proxy-" + msg.RcptTo + "-" + msg.Node
+	sess := globals.sessionStore.Get(sid)
+	if msg.SessGone {
+		// Original session has disconnected. Tear down the local proxied session.
```

Comment:
Previously it meant that the session at origin got disconnected and proxied session at master must be deleted too. What does it mean here? The comment is probably incorrect too.

Reply:
Oh, I see it's an indicator that proxied topic is gone. Maybe rename it accordingly? The comment certainly needs to be updated.